### ISSUE TITLE
Bevformer temporal and no temporal configs are the same

### ISSUE THREAD

**Issue by jasonkhadka** [Daniel-xsy/RoboBEV#6]
Following configs are the same. How did you test non-temporal model?
Temporal config:
```
zoo/BEVFormer/projects/configs/bevformer/bevformer_base.py
```
Non-temporal configs:
```
zoo/BEVFormer/projects/configs/bevformer/bevformer_base_no_temp.py
```

**Comment by Daniel-xsy** [comment]
I've corrected it. You only need to change the following argument to switch between temporal and non-temporal mode.
```
video_test_mode=False
```
Thanks for pointing out this issue.

**Comment by jasonkhadka** [comment]
@Daniel-xsy  Isn't that only for testing? The training is still done with temporal information with Temporal Attention?

**Comment by Daniel-xsy** [comment]
Yes, that's correct. In the official BEVFormer paper, the authors trained two versions: BEVFormer and BEVFormer-S. The BEVFormer-S is trained without temporal information, i.e., ```video_test_mode=False``` for both training and testing. You can find more details [here](https://github.com/fundamentalvision/BEVFormer/issues/53).

However, since we couldn't find the training configuration and checkpoint of BEVFormer-S, we opted to use the BEVFormer checkpoint and turn off `video_test_mode` during testing. This might cause a discrepancy between training and testing, as evident from the lower performance of BEVFormer-non-temporal compared to the official results. We might try to reproduce this version  and update the checkpoint in the future if possible.
